src/components/Layout/MenuStyles.js

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+
+import { background, blue, pink } from '@utils/theme';
+
+export const Logo = styled.img`
+  padding: 6px;
+`;
+
+export const MobileNav = styled.div`
+  background: ${background};
+  bottom: 0;
+  box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+  display: flex;
+  flex-wrap: nowrap;
+  font-size: 12px;
+  justify-content: space-around;
+  line-height: 42px;
+  position: fixed;
+  width: 100%;
+
+  a {
+    padding: 0 6px;
+
+    &:after {
+      top: 0;
+      left: 6px;
+      right: 6px;
+    }
+  }
+`;
+
+export const Space = styled.span`
+  flex-grow: 1;
+`;
+
+export const Wrapper = styled.menu`
+  background: ${background};
+  display: flex;
+  height: 64px;
+  padding: 0 calc((100vw - 1440px) / 2);
```

Comment:
Wait is this different from
```
max-width: 1440px;
margin: auto;
```
? Does that not work? This is a nit though, I'm just curious

Reply:
Uh yeah it would, think I originally had it adjusting slightly instead of just `vw` and `px`, and didn't think to remove the `calc` afterwards 😬